validator/validators/basic/basic_validator.go

```diff
@@ -80,12 +82,63 @@ func (s *BasicValidator) CheckData(startTime, endTime time.Time) error {
 				Value: aws.String(dimension.Value),
 			})
 		}
-		err := s.ValidateMetric(metric.MetricName, metricNamespace, metricDimensions, metric.MetricValue, metric.MetricSampleCount, startTime, endTime)
+		environmentValue := "Generic"
+		operationValue := "operation"
+		serviceValue := "service-name"
+
+		appSignalDimensions := []cwtypes.Dimension{
```

Comment:
these dimensions are already defined in `parameters.yaml`, so you can just use the existing logic to build a dim set? Just omit `InstnanceId` when metric name is for appsignals.